drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb

```diff
@@ -620,6 +674,224 @@ def self.sections(report) # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticC
             ! item.enrollment_coc.in?(item.project_coc_codes)
           },
         },
+        hoh_income_percent_ami: {
+          title: 'Head of Household is Missing Income as a Percent of AMI',
+          description: 'Income as a Percent of AMI is missing or not collected',
+          required_for: 'HoH in HP or RRH',
+          detail_columns: default_detail_columns + [
+            :percent_ami,
+          ],
+          denominator: ->(item) {
+            hoh?(item) && [
+              HudUtility2024.performance_reporting[:rrh],
+              HudUtility2024.performance_reporting[:prevention],
+            ].flatten.include?(item.project_type)
+          },
+          limiter: ->(item) {
+            return false unless hoh?(item)
+            return false unless [
+              HudUtility2024.performance_reporting[:rrh],
+              HudUtility2024.performance_reporting[:prevention],
+            ].flatten.include?(item.project_type)
+
+            item.percent_ami.blank?
```

Comment:
this needs to check for `99` as well, which is a valid value, but indicates it's missing.  Maybe better to check for it being in an accepted value.

```
! item.percent_ami.in?(HudUtility2024.percent_amis.keys - [99])
```
or something like that.

We could adjust the description, to say missing, not collected, or not an accepted value.